megamek/src/megamek/common/loaders/BLKFile.java

```diff
@@ -81,18 +81,21 @@ protected void loadEquipment(Entity t, String sName, int nLoc)
         if (saEquip[0] != null) {
             for (String s : saEquip) {
                 String equipName = s.trim();
+                boolean isOmniMounted = false;
```

Comment:
Is this ever used or just assigned?

Reply:
Did you actually look at the rest of the code?

Reply:
I did, and saw it assigned and not used.

I did miss that it was a previously existing variable, hence the question.

Reply:
It's used in a part of the code that wasn't modified for this PR. Hence my question about actually looking at the rest of the code.

Reply:
I scan read, so I can miss stuff because of it.
I always take a look at the rest of the code when reviewing, to try to avoid dumb mistakes like this. My apologies on this one